### ISSUE TITLE
The posts should not be varchar(255), allow any length

### ISSUE THREAD

**Issue by dmitrinesterenko** [dmitrinesterenko/blog#71]
```
ERROR (string_data_right_truncation): value too long for type character varying(255)
```

Full error here:

ERROR (string_data_right_truncation): value too long for type character varying(255)
web/controllers/writer_controller.ex
14      |> render("new.html")
15  end
16
17  def create(conn, params) do
18   post = Post.changeset(%Post{}, %{title: params["title"], slug: params["slug"], body: params["body"]})
19   {status, response} = Repo.insert(post)
20   case status do
21     :ok -> conn
22       |> redirect(to: "/")
23       |> halt()
24     _ -> conn
blog_phoenix BlogPhoenix.WriterController.create/2
 Show all frames
 web/controllers/writer_controller.ex:19 BlogPhoenix.WriterController.create/2
 web/controllers/writer_controller.ex:1 BlogPhoenix.WriterController.action/2
 web/controllers/writer_controller.ex:1 BlogPhoenix.WriterController.phoenix_controller_pipeline/2
 lib/blog_phoenix/endpoint.ex:1 BlogPhoenix.Endpoint.instrument/4

**Comment by dmitrinesterenko** [comment]
Failing test case:

As you listen to the neighbors rattle with the garbage lids like raccoons you struggle to hear a melody. It's like classical music however it's supposed to make you smarter. \r\n\r\nGarbage lids are meant to make you more apt to write a modern piece of music.\r\n\r\nYou'll see them again in the morning, hellos exchanged while you grab the first distraction. The lids handles heating up to skin searing.\r\n\r\n

**Comment by dmitrinesterenko** [comment]
Fixed in #76
